python/device/device.py

```diff
@@ -786,12 +807,21 @@ def dispatch_incoming_message_thread(self, worker_thread_info):
                 ):
                     # We only inspect messages that have `thief/runId` and `thief/serviceInstanceId` set to the expected values
                     cmd = thief[Fields.CMD]
-                    if cmd == Commands.SERVICE_ACK_RESPONSE:
+                    if cmd in [Commands.SERVICE_ACK_RESPONSE, Commands.METHOD_RESPONSE]:
```

Comment:
I feel like the definition of service_ack is starting to become unclear now. For example, if the command is `Commads.METHOD_RESPONSE`, does this mean that we aren't considering the message we receive back as a service ack?

Reply:
I agree 100%.  I want to rename this as part of my refactor.  Maybe op_id?  I don't konw.